tests/baselines/reference/callWithSpread.js

```diff
@@ -136,8 +136,7 @@ var D = /** @class */ (function (_super) {
     __extends(D, _super);
     function D() {
         var _this = _super.call(this, 1, 2) || this;

```

Comment:
Is it intentional to leave this unused variable behind? I guess it's harmless?

Reply:
Yes and no. It's intentional in that the conditions for the simplification that elides `var _this =` isn't met. We could change the rules, in the future if necessary, but I don't want to spend too much time processing these simplifications during emit, which are primarily only to improve readability when debugging unmapped outputs, so I've generally opted for more coarse-grained rules.